src/docs/implementations/dynatrace.adoc

```diff
@@ -202,12 +206,11 @@ DynatraceConfig dynatraceConfig = new DynatraceConfig() {
 MeterRegistry registry = new DynatraceMeterRegistry(dynatraceConfig, Clock.SYSTEM);
 ----
 
-`DynatraceConfig` is an interface with a set of default methods. If, in the implementation of `get(String k)`, rather than returning `null` you instead bind it to a property source, you can override the default configuration. For example, Micrometer's Spring Boot support binds properties prefixed with `management.metrics.export.dynatrace` directly to the `DynatraceConfig`:
-
```

Comment:
> If, in the implementation of `get(String k)`, rather than returning `null` you instead bind it to a property source, you can override the default configuration.

Was this part intentionally not moved up?

Reply:
Yes, I updated the text block and removed this line since it's an implementation detail and I think it does more to confuse users who read this as one of the first things in the documentation (now that its on top) then it does to clarify things.